src/partner/ducks/eventOrders.ts

```diff
@@ -1,10 +1,11 @@
 import { IOrder } from "../models/Order";
 import { EventService } from "../services";
+import { IFluxStandardAction, IFluxStandarDispatch } from ".";
```

Comment:
rename `IFluxStandarDispatch` to `ReduxDispatch` and import it from `./index.ts`